### PR TITLE
Setup the thoth-station lab repo as per new standards

### PR DESCRIPTION
**Pull request by harshad16**
Setup the thoth-station lab repo as per new standards
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

## Related Issues and Dependencies

Related-to: https://github.com/thoth-station/nepthys/issues/80#issuecomment-998178510

### CODE REVIEW

**Comment by harshad16** [comment]
cc: @fridex 

This pr is has pre-commit changes, requirements-updates and ci updates.

File: thoth/lab/__init__.py [hunk 1]
```diff
@@ -6,3 +6,10 @@
 
 __title__ = "thoth-lab"
 __version__ = "0.2.6"
+
+
+__all__ = [
+    obtain_location,
```
Comment (fridex) [inline thoth/lab/__init__.py:12]:
```suggestion
    obtain_location.__name__,
```

File: thoth/lab/__init__.py [hunk 2]
```diff
@@ -6,3 +6,10 @@
...
+    GraphQueryResult,
```
Comment (fridex) [inline thoth/lab/__init__.py:13]:
```suggestion
    GraphQueryResult.__name__,
```

File: thoth/lab/__init__.py [hunk 3]
```diff
@@ -6,3 +6,10 @@
...
+    packages_info,
```
Comment (fridex) [inline thoth/lab/__init__.py:14]:
```suggestion
    packages_info.__name__,
```

**Review by fridex — COMMENTED** [review]
Few comments to make the import system work. Otherwise LGTM 👍🏻 Thanks for fixing this 🎖️

**Pull request merged**